pkg/xen-tools/xen-start

```diff
@@ -93,6 +93,7 @@ xl unpause "$ID" || bail "xl unpause failed"
 # now star polling for domain status (11 sec interval) in the background
```

Comment:
Hmm comment says to sleep. Did we have sleep code earlier?

Reply:
I found the old code.
Seems odd that the sleep was removed - we now run a subshell forever which does xl info calls.